dataland-backend-utils/src/main/kotlin/org/dataland/datalandbackendutils/email/BaseEmailBuilder.kt

```diff
@@ -0,0 +1,147 @@
+package org.dataland.datalandbackendutils.email
+
```

Comment:
Is this just moved from the previous place or were there code changes?

Reply:
We originally only had the EmailBuilder for bulk data requests. I copied that into the backend and modified it such that it fits the purpose of building data ownership request mails. Then I extracted the similarities between both email builders into this file. And made a few slight modifications

Reply:
I like the extracted email functionability